system-int-test/test/cmr/system_int_test/search/collection_cloud_hosted_search_test.clj

```diff
@@ -0,0 +1,49 @@
+(ns cmr.system-int-test.search.collection-cloud-hosted-search-test
```

Comment:
you could have put your test here: cmr.system-int-test.search.service.collection-service-search-test  but this is fine too.

Reply:
yea, but that file is already pretty long. I get lost in the huge test files. I had thought about putting the tag test somewhere else, with other tag tests. In the end I decided it was best to keep them together. I also have a habit of running option-cmd-x to run all tests in the namespace.